ifrs17/Import/Importers.ipynb

```diff
@@ -503,18 +503,33 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs args, IDataSource dataSource, string format, IDataSource targetDataSource)",
+                "public async Task<ImportArgs[]> GetAllArgsAsync(ImportArgs args, IDataSource dataSource, string format)",
                 "\n{",
                 "\n    ImportArgs[] allArgs;",
                 "\n    switch(format)",
                 "\n    {",
-                "\n        case ImportFormats.YieldCurve : ",
+                "\n        case ImportFormats.Cashflow:",
+                "\n        case ImportFormats.Actual: ",
+                "\n        case ImportFormats.Opening:",
+                "\n        {",
+                "\n            if(args.Scenario != null) return args.RepeatOnce().ToArray();",
+                "\n            var importFormat = (format == ImportFormats.Actual)? ImportFormats.Actual : ImportFormats.Cashflow;",
```

Comment:
Now I think Opening also imports Advance and Overdue, and you would then need to leave format = opening else with Cashflow you would have the same problem we discussed for actuals.

Reply:
ok done